### PR TITLE
Fix name: astropile vs mmu

### PR DESCRIPTION
**Pull request by mb010**
Removing references to Astropile. Package name changed to `mmu` with folder names changed to MultimodalUniverse.

This may break some local tests - a given user who is actually using those specific paths will simply have to rename local folders. Some ofwhich may already have had their names updated.

Addresses #119

### CODE REVIEW

File: scripts/manga/manga.py [hunk 1]
```diff
@@ -109,8 +109,9 @@ def _info(cls):
         # add the reconstructed image features
         features['images'] = [{
             'filter': Value('string'),
-            'array': Array2D(shape=(cls._image_size, cls._image_size), dtype='float32'),
-            'array_units': Value('string'),
+            'flux': Array2D(shape=(cls._image_size, cls._image_size), dtype='float32'),
+            'flux_err': Array2D(shape=(cls._image_size, cls._image_size), dtype='float32'),
```
Comment (mb010) [inline scripts/manga/manga.py:113]:
```suggestion
```

File: scripts/manga/manga.py [hunk 2]
```diff
@@ -214,10 +215,10 @@ def reshape_if_needed(col_name, data):
                         for i in grp['spaxels']
                     ]
 
-                    im_cols = ('filter', 'array', 'array_units', 'psf', 'psf_units', 'scale', 'scale_units')
+                    im_cols = ('filter', 'flux', 'flux_err', 'flux_units', 'psf', 'psf_units', 'scale', 'scale_units')
```
Comment (mb010) [inline scripts/manga/manga.py:218]:
```suggestion
                    im_cols = ('filter', 'flux', 'flux_units', 'psf', 'psf_units', 'scale', 'scale_units')
```

File: scripts/desi/desi.py [hunk 3]
```diff
@@ -51,21 +51,21 @@
...
+    "flux_z",
+    "flux_ivar_g",
+    "flux_ivar_r",
+    "flux_ivar_z",
+    "fiberflux_g",
+    "fiberflux_r",
+    "fiberflux_z",
+    "fibertotflux_g",
+    "fibertotflux_r",
+    "fibertotflux_z",
```
Comment (mb010) [inline scripts/desi/desi.py:68]:
```suggestion
    "EBV",
    "FLUX_G",
    "FLUX_R",
    "FLUX_Z",
    "FLUX_IVAR_G",
    "FLUX_IVAR_R",
    "FLUX_IVAR_Z",
    "FIBERFLUX_G",
    "FIBERFLUX_R",
    "FIBERFLUX_Z",
    "FIBERTOTFLUX_G",
    "FIBERTOTFLUX_R",
    "FIBERTOTFLUX_Z",
```

**Review by mb010 — COMMENTED** [review]
Field name corrections of incorrect changes.

**Review by LTMeyer — APPROVED** [review]
LGTM

**Pull request merged**